src/components/TopBar/Annotations.js

```diff
@@ -48,6 +48,46 @@ export const Annotations = observer(({ store, annotationStore }) => {
     return () => document.removeEventListener('click', handleClick);
   }, []);
 
+  const renderAnnotation = (ent, i) => {
+    return (
+      <Annotation
+        key={`${ent.pk ?? ent.id}${ent.type}`}
+        entity={ent}
+        aria-label={`${ent.type} ${i + 1}`}
+        selected={ent === annotationStore.selected}
+        onClick={e => {
+          e.preventDefault();
+          e.stopPropagation();
+          setOpened(false);
+          onAnnotationSelect?.(ent, ent.type === 'prediction');
+        }}
+        extra={groundTruthEnabled && (
+          <GroundTruth entity={ent} disabled/>
+        )}
+      />
+    );
+  };
+
+  const renderAnnotationList = (entities) => {
+    const _drafts = [];
+    const _annotations = [];
+
+    entities.forEach((obj, i) => {
+      if (obj.pk) {
+        _annotations.push(renderAnnotation(obj, i));
+      } else {
+        _drafts.push(renderAnnotation(obj, i));
+      }
+    });
+
+    return (
+      <>
+        <Elem name={'draft'}>{_drafts}</Elem>
+        <Elem name={'annotation'}>{_annotations}</Elem>
```

Comment:
Minor tweak tot he names as they are just a string
```suggestion
        <Elem name="draft">{_drafts}</Elem>
        <Elem name="annotation">{_annotations}</Elem>
```